Geometry/HGCalCommonData/src/HGCalDDDConstants.cc

```diff
@@ -786,10 +786,10 @@ bool HGCalDDDConstants::isValidTrap(int zside, int layer, int irad, int iphi) co
   bool valid = ((ok && trapezoidFile()) ? tileExist(zside, layer, irad, iphi) : ok);
   bool tileEx = trapezoidFile() ? tileExist(zside, layer, irad, iphi) : true;
   if (!valid)
-    edm::LogWarning("HGCalGeomT") << "HGCalDDDConstants::isValidityTrap: Input " << zside << ":" << layer << ":" << irad
-                                  << ":" << iphi << " Range on Ring " << hgpar_->scintValidRing(indx.first, irad)
-                                  << " Range on phi 0:" << hgpar_->scintCells(layer) << " tileExist " << tileEx
-                                  << " Valid " << ok << ":" << tileExist(zside, layer, irad, iphi) << ":" << valid;
+    edm::LogVerbatim("HGCalGeomT") << "HGCalDDDConstants::isValidityTrap: Input " << zside << ":" << layer << ":" << irad
```

Comment:
I think this should really be a LogDebug level information and not printed at all in production jobs.

Reply:
I think it was not a debug printing. This is an unusual case

Reply:
`LogVerbatim` is excessive, please demote it to something that does not get printed by defaul.